ssg/build_profile_remediations.py

```diff
@@ -69,6 +69,10 @@ def get_output_paths(benchmarks, benchmark_profile_pairs, path_base, extension,
     paths = []
 
     for benchmark_id, profile_id, _ in benchmark_profile_pairs:
+        # profile (default) is not applicable when generating remediation
+        if profile_id == "":
```

Comment:
It's not a big deal, but the so-called Pythonic way is to write `if not profile_id`: https://www.python.org/dev/peps/pep-0008/#programming-recommendations

Reply:
I agree, but in other parts of the code it does this way, I guess the intention of the original author was to make explicit that it is the `default` profile when it is an empty string. But I'm fine changing that.

Reply:
I replaced every occurrence of `profile_id == ""`